vaadin-dialog-flow-integration-tests/src/test/java/com/vaadin/flow/component/dialog/tests/DialogTestPageIT.java

```diff
@@ -205,6 +205,34 @@ public void openEmptyDialog_dialogContentHasWidth() {
                 greaterThan(endpoint));
     }
 
+    @Test
+    public void verifyDialogFullSize() {
+        findElement(By.id("button-for-dialog-with-div")).click();
+        WebElement overlay = getInShadowRoot(getOverlayContent(),
+                By.id("overlay"));
+        Assert.assertTrue(
+                overlay.getAttribute("style").contains("width: 100%;"));
+        Assert.assertTrue(
+                overlay.getAttribute("style").contains("height: 100%;"));
+
+        WebElement div = findElement(By.id("div-in-dialog"));
+        WebElement content = overlay.findElement(By.id("content"));
+
+        String overLayWidth = overlay.getCssValue(ElementConstants.STYLE_WIDTH);
+        int overlayValue = Integer
+                .valueOf(overLayWidth.substring(0, overLayWidth.length() - 2));
+
+        String paddingWidth = content.getCssValue("padding");
+        int paddingValue = Integer
+                .valueOf(paddingWidth.substring(0, paddingWidth.length() - 2));
+
+        String divWidth = div.getCssValue(ElementConstants.STYLE_WIDTH);
+        int divValue = Integer
+                .valueOf(divWidth.substring(0, divWidth.length() - 2));
+
+        Assert.assertEquals(overlayValue - paddingValue * 2, divValue);
```

Comment:
We need to run the same tests for height.

Note: I wouldn't guess from this line what "overlayValue" means. Rename to "overlayWidthValue" or something

Reply:
Done.